app/src/main/res/layout/item_autocomplete_emoji.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please change `RelativeLayout` to be something else please (`LinearLayout` or `ConstraintLayout`). Ask for help if needed.